src/common/libzmqutil/cert.h

```diff
@@ -0,0 +1,37 @@
+/************************************************************\
+ * Copyright 2023 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef _ZMQUTIL_CERT_H
+#define _ZMQUTIL_CERT_H
+
+#include <stdio.h>
+#include <stdbool.h>
+
+struct cert *zmqutil_cert_create (void);
```

Comment:
It feels a little strange that this class is `zmqutil_cert` but the structure is just `struct cert`. However, it probably doesn't rise to the level of updating the whole PR. (just a comment)

Reply:
Yeah and it's a narrowly used internal class so maybe just claiming the "cert_" namespace is OK here.  I can make that change.